.circleci/config.yml

```diff
@@ -255,6 +255,27 @@ jobs:
       - *build-Example
       - *save-cache
 
+  spm-integration-test:
+    parameters:
+      iOS:
+        type: string
+        default: "14.4"
+      xcode:
+        type: string
+        default: "12.4.0"
+    macos:
+      xcode: << parameters.xcode >>
+    environment:
+      HOMEBREW_NO_AUTO_UPDATE: 1
+    steps:
+      - checkout
+      - *prepare-mapbox-file
+      - *prepare-netrc-file
+      - *add-github-to-known-hosts
+      - run: sed -i '' -e "s/6cc9728a615ed9b08632a6c2d09f7fca57a5c542/`git rev-parse --verify HEAD`/" Tests/SPMTest/SPMTest.xcodeproj/project.pbxproj # Replace SPM package reference with current revision
```

Comment:
Should we factor this command into a script to allow verifying locally on the command line?

Reply:
Sure, I added a new script `scripts/build_spm_integration_test.sh` in this PR.